clusterdock/models.py

```diff
@@ -212,6 +212,9 @@ class Node:
             Default: ``None``
         devices (:obj:`list`, optional): Devices on the host to expose to the node. Default:
             ``None``
+        privileged (:obj:`bool`, optional): Give extended privileges to the container. Default: ``False``
+        sync_localtime_by_mount (:obj:`bool`, optional): Sync host time to Docker container by either /etc/localtime or
```

Comment:
Is this really something that would differ between nodes in the cluster? Perhaps better to have this set at the cluster level? Or perhaps don't even make it an arg like this and have it as a module-level variable that could be set by topologies (e.g. `from clusterdock.models import LOCALTIME_MOUNT; LOCALTIME_MOUNT = False`)